### PR TITLE
Personality Captions Human Eval Task

### PR DESCRIPTION
**Pull request by klshuster**
Task that shows two comments for one image (and optionally the personality) and asks the turker to 1) choose the more engaging comment and 2) explain their choice. 

The current setup just lists the collected comment from the Personality-Captions dataset twice; if one were to provide an `--eval-data-path`, the task would use this for evaluation.

While the PR claims there are over 800 additions, the _meaningful_ code snippets are relatively short.

cc @jaseweston

### CODE REVIEW

File: parlai/mturk/tasks/personality_captions/stack_rank_evals/run.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        Human Evaluation of various image captions/comments.
+
+        A turker is shown an image and two possible comments/captions, and
+        optionally the personality used to create these captions. Then, the
+        turker is asked to choose which caption they think is more engaging.
+
+        In this example, we will just be comparing the original comment twice
+        (this is just to demonstrate the task for future use).
+
+        To use your own data, please specify `--eval-data-path` to an
```
[the remark this answers is not in the document]
Comment (klshuster) [reply parlai/mturk/tasks/personality_captions/stack_rank_evals/run.py]: Yeah I'll include an example, that's a good idea. Basically, the goal is that each example has a bunch of dict keys that map to things you'd want to compare. For example, suppose you wanted to compare the original comment and a comment generated by a model called `happy_model`. You might have data that looks like this:

`[{'image_hash': iamahash1, 'personality': 'sweet', 'comment': 'Nice!', 'happy_model_comment': 'I am so happy!'}, {'image_hash':...}...]`

You would use the following command:

`python run.py -ck1 comment -ck2 happy_model_comment --eval-data-path /path/to/data`

**Comment by klshuster** [comment]
Perhaps later I will port this task over to react to help myself and others

**Comment by JackUrb** [comment]
@klshuster Once I finish the new react templates (2 col with context in bottom of left col, 3 col with context in middle) it should be pretty straightforward to port.

**Comment by klshuster** [comment]
I've added an example to the main docstring (see response to @JackUrb comment; it's very similar)

**Pull request merged**